planner/matchmaker/pom.xml

```diff
@@ -30,8 +30,35 @@ limitations under the License.
     <dependencies>
         <dependency>
             <groupId>eu.seaclouds-project</groupId>
-            <artifactId>toscamodel</artifactId>
-            <version>1.0.0</version>
```

Comment:
I guess that the toscamodel repository can be now removed from planner pom, right?
(https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/planner/pom.xml#L50)